notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.        plt.savefig(f"{plot_name}.pgf", backend="pgf")
</pre><p>Could leave this as responsibility of callee and remove plot_name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>